src/components/VLoadMore.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <VButton
+    size="large"
+    variant="full"
+    type="button"
+    :disabled="isFetching"
+    @click="onLoadMore"
+  >
+    {{ buttonLabel }}
+  </VButton>
+</template>
+<script>
+import { computed, defineComponent, useContext } from '@nuxtjs/composition-api'
+export default defineComponent({
+  name: 'VLoadMore',
+  props: {
+    isFetching: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  setup(props, { emit }) {
+    const { i18n } = useContext()
+
+    const buttonLabel = computed(() => {
```

Comment:
This could be turned into a fully-presentational component if we just passed the translated label as a prop, but that might be going too far :)